src/pages/Share.tsx

```diff
@@ -21,7 +22,13 @@ const Share = () => {
   };
 
   const handleDeepLink = () => {
-    window.location.replace('instagram-stories://share');
+    if (isIOS) {
+      window.location.replace('instagram-stories://share');
+    } else if (isAndroid) {
+      window.location.replace('intent://instagram.com/#Intent;scheme=https;package=com.instagram.android;end');
+    } else {
+      window.location.replace('https://www.instagram.com/');
+    }
```

Comment:
오오 이거 이렇게 하는거구나!이거 if ,else if 말고 
```suggestion
if (isIOS) {
      return window.location.replace('instagram-stories://share');
    } 
    if (isAndroid) {
   return window.location.replace('intent://instagram.com/#Intent;scheme=https;package=com.instagram.android;end');
    } 
      return window.location.replace('https://www.instagram.com/');
    
```
이케는 안데나?!

Reply:
> 오오 이거 이렇게 하는거구나!이거 if ,else if 말고
> 
> 이케는 안데나?!

효율을 위해서는 제한해준 방식이 더 좋은 것 같어 리팩토링 과정에서 적용해갈게~~